samples/client/petstore/go/go-petstore/enum_class.go

```diff
@@ -10,5 +10,12 @@
 
 package petstore
 
-type EnumClass struct {
-}
+type EnumClass string
+
+// List of EnumClasss
+const (
+	_abc EnumClass = "_abc"
```

Comment:
Please have a look at https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/resources/php/model_generic.mustache#L69-L84

You can also use the `-DdebugModels` flag to find out what mustache tags are available: https://github.com/swagger-api/swagger-codegen/wiki/FAQ#how-to-debug-swagger-codegen